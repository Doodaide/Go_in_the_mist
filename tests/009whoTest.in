who
who
place A6
who
who
place B12
history
view
place C3
who
resign