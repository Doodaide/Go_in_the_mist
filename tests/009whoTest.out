B
B
W
W
A6B12
L8,.................................................
W
Black wins!
A6B12C3
Thank you for playing!
